Core/Base/ConsumerReactor.py

```diff
@@ -0,0 +1,61 @@
+"""
+  DIRAC class to execute consumers
+
+  All DIRAC consumers must inherit from the basic class ConsumerModule
+
+  The class is created based on the AgentReactor example.
+  In the most common case, DIRAC consumers are executed using the dirac-consumer command.
+  dirac-consumer accepts a list positional arguments. These arguments have the form:
+  [DIRAC System Name]/[DIRAC Consumer Name]
+  dirac-consumer then:
+  - produces a instance of ConsumerReactor
+  - loads the required module using the loadModule method
+  - starts the consumer itself using the ConsumerReactor.go method
+
+"""
+
+from DIRAC.Core.Base.ConsumerModule import ConsumerModule
+from DIRAC.Core.Base.private.ModuleLoader import ModuleLoader
+from DIRAC.WorkloadManagementSystem.Consumer.ConsumerTools import getConsumerSection
+from DIRAC import S_OK, DError
```

Comment:
Fixed in the next push.
Btw his module  should be in the second PR so I will move it there.